log/logger.go

```diff
@@ -169,6 +169,32 @@ func joinLevelRanks(levelRanks ...LevelRanks) LevelRanks {
 	return joined
 }
 
+func addPrefix(fields ...Fields) Fields {
+	joined := Fields{}
+	for _, inner := range fields {
+		for key, value := range inner {
+			if key != "" {
+				if value != nil {
+					prefixedKey := "dbl_" + key
```

Comment:
would that make sense to have the 'dbl_' in a constant, so that we can juste use it in the different impacted files?